crates/noirc_evaluator/src/ssa/ir/instruction.rs

```diff
@@ -140,7 +140,7 @@ pub(crate) enum Instruction {
     Truncate { value: ValueId, bit_size: u32, max_bit_size: u32 },
 
     /// Constrains a value to be equal to true
-    Constrain(ValueId),
+    Constrain(ValueId, ValueId),
```

Comment:
This comment can be updated, we now constrain these two values to be equal.

Reply:
Thanks, addressed this.